ui/page/transaction/transactions_page.go

```diff
@@ -101,50 +115,42 @@ func (pg *TransactionsPage) OnNavigatedTo() {
 	go pg.scroll.FetchScrollData(false, pg.ParentWindow())
 }
 
-func (pg *TransactionsPage) sectionNavTab(gtx C) D {
-	var selectedTabDims D
-
-	return layout.Inset{
-		Top: values.MarginPadding20,
-	}.Layout(gtx, func(gtx C) D {
-		return pg.tabs.Layout(gtx, len(txTabs), func(gtx C, i int) D {
-			return layout.Stack{Alignment: layout.S}.Layout(gtx,
-				layout.Stacked(func(gtx C) D {
-					return layout.Inset{
-						Right:  values.MarginPadding24,
-						Bottom: values.MarginPadding14,
-					}.Layout(gtx, func(gtx C) D {
-						return layout.Center.Layout(gtx, func(gtx C) D {
-							lbl := pg.Theme.Label(values.TextSize16, txTabs[i])
-							lbl.Color = pg.Theme.Color.GrayText1
-							if i == pg.selectedTabIndex {
-								lbl.Color = pg.Theme.Color.Primary
-								selectedTabDims = lbl.Layout(gtx)
-							}
+// initWalletSelector is used by the tx history page, for wallet selection.
+func (pg *TransactionsPage) initWalletDropdown() {
```

Comment:
This is the tx history page, what else would use this method!?
```suggestion
// initWalletSelector initializes the wallet selector dropdown to enable filtering
// transactions for a specific wallet when this page is used to display transactions
// for multiple wallets.
func (pg *TransactionsPage) initWalletDropdown() {
```

Reply:
This comment reminds me, I hope we don't show a dropdown if there's only one wallet in the app. If there's only one wallet, it should be auto selected and the page should render as if it were displaying transactions for just one wallet, which it is.

Reply:
It auto selects the first wallet in the dropdown, but does not hide the wallet selector is its just one.

Reply:
Fixed